core/src/scene/drawRule.h

```diff
@@ -37,11 +38,10 @@ class StyleContext;
 struct DrawRuleData {
 
     std::vector<StyleParam> parameters;
-    std::string name;
-    int id;
+    std::string styleName;
+    int styleId;
```

Comment:
It seems the tangram-docs need to be updated to state that ```draw-style``` may be an arbitrary string when a ```style``` parameter is provided.
In my opinion styleName is not that bad - Either draw-style references a style name or it implicitly defines a new style name (inheriting parameters from a predefined style)
Or ```name``` member  could be renamed to ```drawStyle``` for matching the docs 1-to-1

https://github.com/tangrams/tangram-docs/blob/gh-pages/pages/draw.md#draw-style
https://github.com/tangrams/tangram-docs/blob/gh-pages/pages/draw.md#style